microsoft-edge/toc.yml

```
@@ -1013,89 +1034,141 @@
               href: extensions-chromium/developer-guide/sidebar.md
               displayName: sidebar extensions, sidepanel, side panel  # alt lookup phrase
 
-            - name: Best Practices
-              href: extensions-chromium/developer-guide/best-practices.md
-              displayName: Best Practices for extensions  # full title
+#           - name: Microsoft Edge Add-ons API  # It will redirect to all about the APIs page
+#             href: 
 
-        - name: Manifest
+# -------------------------------------
+        - name: Use extension APIs
```

```suggestion
        - name: Use REST APIs for extensions
```
@sivMSFT -- @captainbrosset elsewhere recommended specifying/clarifying that we're not talking about eg win32 APIs, but REST APIs

---

changed toc bucket name from:
Use extension APIs
to:
REST API for extensions

@sivMSFT